ldb.go

```diff
@@ -34,6 +35,14 @@ func init() {
 // ReaderForPath opens an LDB at the provided path and returns an LDBReader
 // instance pointed at that LDB.
 func ReaderForPath(path string) (*LDBReader, error) {
+	_, err := os.Stat(path)
+	switch {
```

Comment:
Could probably just return `err` here without checking the exact kind of error, which would print something like "/does/not/exist/ldb.db: No such file or directory" for `IsNotExist`.